datasets/chorus-dt/utils/dataset.py

```diff
@@ -48,26 +54,47 @@ def get_places_and_trips(data, prestation_types=None):
         current_data["lieu_arrivee"].value_counts().to_frame().reset_index()
     )
     lieu_arrivee_count.columns = ["place", "dst"]
+    
     lieu_etape_count = (
         current_data["lieu_etape"].value_counts().to_frame().reset_index()
     )
     lieu_etape_count.columns = ["place", "stop"]
-    print(len(lieu_arrivee_count))
-    print(len(lieu_depart_count))
+
     places = pd.merge(lieu_depart_count, lieu_arrivee_count, how="outer", on="place")
     places = pd.merge(places, lieu_etape_count, how="outer", on="place")
     places.fillna(0, inplace=True)
     places["total"] = places["src"] + places["dst"] + places["stop"]
+    
+    all_trips_w_etape = pd.DataFrame()
+    current_data_w_etape = current_data[~current_data['lieu_etape'].isna()]
 
-    all_trips = pd.DataFrame()
-    places_columns = ["lieu_depart", "lieu_etape", "lieu_arrivee"]
-    all_trips["trip_slug"] = current_data[places_columns].apply(
-        lambda x: " <-> ".join(x), axis=1
-    )
-    all_trips["trip_place_0"] = current_data["lieu_depart"]
-    all_trips["trip_place_1"] = current_data["lieu_etape"]
-    all_trips["trip_place_2"] = current_data["lieu_arrivee"]
-    all_trips["prestation_type"] = current_data["prestation_type"]
+    if not current_data_w_etape.empty:
```

Comment:
It might probably be done in a more compact way, like:
```suggestion
all_trips["trip_place_0"] = current_data["lieu_depart"] 
all_trips["trip_place_1"] = current_data["lieu_etape"]
all_trips["trip_place_2"] = current_data["lieu_arrivee"]
         
all_trips.loc[current_data["lieu_etape"].isna(), "trip_place_1"] = current_data["lieu_arrivee"]
all_trips.loc[current_data["lieu_etape"].isna(), "trip_place_2"] = "No stop"
```